OpenRA.Mods.Common/Traits/Render/RenderHarvester.cs

```diff
@@ -16,7 +16,9 @@ namespace OpenRA.Mods.Common.Traits
 {
 	class RenderHarvesterInfo : RenderUnitInfo, Requires<HarvesterInfo>
 	{
+		/* [SequenceReference] TODO: lint rule does not support arrays */
```

Comment:
Yup.  The comment here was first, and applied specifically to the context of the harvester sprites.  The other places are best represented with an array of images, so this is a problem for them.